src/Aspire.ProjectTemplates.9.0.net9/templates/aspire-servicedefaults/Aspire.ServiceDefaults1.csproj

```diff
@@ -0,0 +1,22 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net8.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+    <IsAspireSharedProject>true</IsAspireSharedProject>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <FrameworkReference Include="Microsoft.AspNetCore.App" />
+
+    <PackageReference Include="Microsoft.Extensions.Http.Resilience" Version="!!REPLACE_WITH_EXTENSIONS_VERSION!!" />
```

Comment:
Do we need to have separate replacement tokens for extensions 8.x vs. 9.x, so that the `net8.0`-targeting template brings in 8.x versions of extensions, and the `net9.0`-targeting template brings in 9.x versions of extensions?

/Cc @eerhardt

Reply:
@DamianEdwards, I imagine that would be the case, as I believe I did see an issue with that when testing the templates.

Reply:
done.